### PR TITLE
Upgrade Azure Storage to 5.0

### PR DESCRIPTION
**Pull request by jason-bragg**

### CODE REVIEW

File: src/OrleansAzureUtils/OrleansAzureUtils.csproj [hunk 1]
```diff
@@ -61,29 +61,38 @@
...
-      <HintPath>$(SolutionDir)packages\Microsoft.Data.Edm.5.6.0\lib\net40\Microsoft.Data.Edm.dll</HintPath>
+    <Reference Include="Microsoft.Azure.KeyVault.Core, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\packages\Microsoft.Azure.KeyVault.Core.1.0.0\lib\net40\Microsoft.Azure.KeyVault.Core.dll</HintPath>
+      <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.Data.OData">
-      <HintPath>$(SolutionDir)packages\Microsoft.Data.OData.5.6.0\lib\net40\Microsoft.Data.OData.dll</HintPath>
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="Microsoft.Data.Edm, Version=5.6.4.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\packages\Microsoft.Data.Edm.5.6.4\lib\net40\Microsoft.Data.Edm.dll</HintPath>
```
Comment (gabikliot) [inline src/OrleansAzureUtils/OrleansAzureUtils.csproj]: I suspect @jthelin will (justifiably) give you a hard time with those hard coded paths.

Comment (sergeybykov) [inline src/OrleansAzureUtils/OrleansAzureUtils.csproj]: Not path, version.

Comment (jthelin) [inline src/OrleansAzureUtils/OrleansAzureUtils.csproj]: Yes, fully qualified assembly names with version numbers in them are E.V.I.L. ;)

**Comment by sergeybykov** [comment]
@jason-bragg Let's remove the version info, rebase, and merge after that.

File: src/OrleansAzureUtils/OrleansAzureUtils.csproj (hunk 1 above)
Comment (gabikliot) [inline src/OrleansAzureUtils/OrleansAzureUtils.csproj]: We worked too long together @jthelin :-)

**Comment by jason-bragg** [comment]
Holding this PR until I workout the azure queue stream testing issues.  I want to make sure we've a clean test harness to vet the upgrade.  Not pulling the PR, just not updating until our tests are solid.

**Comment by jason-bragg** [comment]
Updated.
Tested.
Rebased.
Squashed.

**Pull request merged**